### PR TITLE
Add direct dashboard access button to changelog

### PR DESCRIPTION
**Pull request by Steve-Mcl**
## Description

Add a changelog entry to accompany "Direct Access to Node-RED Dashboard from FlowFuse UI" #3611

## Related Issue(s)

https://github.com/FlowFuse/flowfuse/issues/3611

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

File: src/changelog/2024/04/device-auto-snapshot.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+title: Direct Dashboard Access
+date: 2024-03-02 13:00:00.0
+authors: ["stephen-mclaughlin"]
+tags:
+    - changelog
+---
+
+We have added quick access buttons to help you navigate to your shiny Dashboard in a single click.
```
Comment (joepavitt) [inline src/changelog/2024/04/device-auto-snapshot.md:9]:
```suggestion
We have added quick access buttons to help you navigate to your Dashboard 2.0 UI in a single click.
```

**Review by joepavitt — APPROVED** [review]
Approve - minor suggestion, feel free to ignore though

**Comment by ZJvandeWeg** [comment]
@joepavitt @Steve-Mcl Other than the timestamp, is this good to go?

**Comment by Steve-Mcl** [comment]
> @joepavitt @Steve-Mcl Other than the timestamp, is this good to go?

It was approved. And I did apply the perfectly good suggestion. Good to go from my PoV.

File: src/changelog/2024/04/device-auto-snapshot.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
+title: Direct Dashboard Access
+date: 2024-03-02 13:00:00.0
```
Comment (ZJvandeWeg) [inline src/changelog/2024/04/device-auto-snapshot.md:3]:
```suggestion
date: 2024-03-06 13:00:00.0
```

**Review by ZJvandeWeg — APPROVED** [review]

**Pull request merged**